src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/cas2/PersonUtils.kt

```diff
@@ -17,8 +17,9 @@ fun OffenderService.getInfoForPersonOrThrowInternalServerError(crn: String):
 fun OffenderService.getFullInfoForPersonOrThrow(crn: String):
   PersonInfoResult.Success.Full {
   val personInfo = this.getInfoForPerson(crn)
-  if (personInfo is PersonInfoResult.NotFound) throw NotFoundProblem(crn, "Offender")
-  if (personInfo is PersonInfoResult.Success.Restricted) throw ForbiddenProblem()
-
-  return personInfo as PersonInfoResult.Success.Full
+  when (personInfo) {
+    is PersonInfoResult.NotFound, is PersonInfoResult.Unknown -> throw NotFoundProblem(crn, "Offender")
+    is PersonInfoResult.Success.Restricted -> throw ForbiddenProblem()
```

Comment:
For CAS2 personInfo will never be PersonInfoResult.Restricted, however the check has been left here in PersonUtils.kt.  Therefore there is also no requirement to add a unit test to test this condition (since it can't be tested since it's never returned).